### PR TITLE
Fix #176: Add Paradox directive to show binding build tool integration

### LINKED ISSUE #176: Fix artifactId of bindings on bintray
Currently artifactId in documentation does not match real one.

For example artifactId for [POSIX bindings](https://bintray.com/scala-native-bindgen/maven/libposix) is `libposix_native0.3` but [docs](https://scala-native.github.io/scala-native-bindgen/bindings/posix.html) say it's `libposix`.
I suppose we should change artifactId on bintray so it doesn't contain version and `_native` postfix.

### PR DESCRIPTION
**Pull request by jonas**
screen shot 2018-09-03 at 22 46 14
screen shot 2018-09-03 at 22 46 21
screen shot 2018-09-03 at 22 46 27

### CODE REVIEW

File: project/ParadoxSupport.scala [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      node.contentsNode.getChildren.asScala.headOption match {
+        case Some(text: TextNode) =>
+          renderBindingDependency(text.getText, printer)
+        case _ => node.contentsNode.accept(visitor)
+      }
+    }
+
+    def renderBindingDependency(binding: String, printer: Printer): Unit = {
+      val artifactName = s"lib$binding"
+      val artifactId   = s"${artifactName}_native0.3_${ScalaBinaryVersion}"
```
Comment (jonas) [inline project/ParadoxSupport.scala]: `native0.3` is currently hard-coded. I don't know if it is possible to get it from the sbt-scala-native plugin.

File: docs/src/paradox/bindings/iconv.md [hunk 2]
```diff
@@ -2,20 +2,7 @@
...
-
-Gradle
-:   @@snip [build.gradle](../resources/build.gradle)
-
-@@dependency[sbt,Maven,Gradle] {
-  group="org.scala-native.bindgen"
-  artifact="libiconv_$scala.binary.version$"
-  version="$project.version$"
-}
+@@binding[iconv]
```
Comment (kornilova-l) [inline docs/src/paradox/bindings/iconv.md]: ❤️

**Comment by kornilova-l** [comment]
Closes #176

**Comment by jonas** [comment]
Thanks for the review, I will work on releasing version 0.3.1.

**Pull request merged**